### PR TITLE
Blog: Camel Quarkus 1.8.0

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the release 1.8.0 of Camel Quarkus! Here are the highlights.
+
+## Camel 3.9.0
+
+Camel 3.9.0 is rich in optimizations. It
+[reduces object allocations](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#reduced-object-allocations),
+[optimizes Camel Core](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#optimized-core) and
+the [HTTP component](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#optimized-http-component).
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:19]: I think you'll need to use site relative links here. The reason being that the link checker doesn't check remote links, so we enforce all links that point to camel.apache.org to be site relative so they can be checked and we can notice broken links.

```suggestion
[reduces object allocations](/blog/2021/03/Camel39-Whatsnew/#reduced-object-allocations),
[optimizes Camel Core](/blog/2021/03/Camel39-Whatsnew/#optimized-core) and
the [HTTP component](/blog/2021/03/Camel39-Whatsnew/#optimized-http-component).
```

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+Camel 3.9.0 further brings
+[optional property placeholders](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#optional-property-placeholders)
+(`file:foo?bufferSize={{?myBufferSize}}`)
+and improvements in Kafka components](https://camel.apache.org/blog/2021/03/Camel39-Whatsnew/#kafka).
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:24]:
```suggestion
[optional property placeholders](/blog/2021/03/Camel39-Whatsnew/#optional-property-placeholders)
(`file:foo?bufferSize={{?myBufferSize}}`)
and improvements in Kafka components](/blog/2021/03/Camel39-Whatsnew/#kafka).
```

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+These are mostly motivated by [Camel K](https://camel.apache.org/camel-k/latest/)
+and its project-less approach to defining Camel routes.
+With Camel K, all you need to run an integration is a file where the routes are defined using one of the supported DSLs
+([Java jOOR](https://camel.apache.org/camel-quarkus/latest/reference/extensions/java-joor-dsl.html),
+[XML](https://camel.apache.org/camel-quarkus/latest/reference/extensions/xml-io-dsl.html),
+[Groovy](https://camel.apache.org/camel-quarkus/latest/reference/extensions/groovy-dsl.html),
+[YAML](https://camel.apache.org/camel-quarkus/latest/reference/extensions/yaml-dsl.html),
+[Kotlin](https://camel.apache.org/camel-quarkus/latest/reference/extensions/kotlin-dsl.html)
+or [JavaScript](https://camel.apache.org/camel-quarkus/latest/reference/extensions/js-dsl.html)).
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:48]:
```suggestion
([Java jOOR](/camel-quarkus/latest/reference/extensions/java-joor-dsl.html),
[XML](/camel-quarkus/latest/reference/extensions/xml-io-dsl.html),
[Groovy](/camel-quarkus/latest/reference/extensions/groovy-dsl.html),
[YAML](/camel-quarkus/latest/reference/extensions/yaml-dsl.html),
[Kotlin](/camel-quarkus/latest/reference/extensions/kotlin-dsl.html)
or [JavaScript](/camel-quarkus/latest/reference/extensions/js-dsl.html)).
```

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+## New extensions
+
+Except for the new DSLs mentioned above, there are six new extensions:
+
+* [Azure Storage Data Lake](https://camel.apache.org/camel-quarkus/latest/reference/extensions/azure-storage-datalake.html) (JVM only)
+* [Freemarker](https://camel.apache.org/camel-quarkus/latest/reference/extensions/freemarker.html) (native)
+* [HL7](https://camel.apache.org/camel-quarkus/latest/reference/extensions/hl7.html) (native)
+* [Huawei Cloud SMN](https://camel.apache.org/camel-quarkus/latest/reference/extensions/huaweicloud-smn.html) (JVM only)
+* [Paho MQTT 5](https://camel.apache.org/camel-quarkus/latest/reference/extensions/paho-mqtt5.html) (native)
+* [Stitch](https://camel.apache.org/camel-quarkus/latest/reference/extensions/stitch.html) (JVM only)
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:61]:
```suggestion
* [Azure Storage Data Lake](/camel-quarkus/latest/reference/extensions/azure-storage-datalake.html) (JVM only)
* [Freemarker](/camel-quarkus/latest/reference/extensions/freemarker.html) (native)
* [HL7](/camel-quarkus/latest/reference/extensions/hl7.html) (native)
* [Huawei Cloud SMN](/camel-quarkus/latest/reference/extensions/huaweicloud-smn.html) (JVM only)
* [Paho MQTT 5](/camel-quarkus/latest/reference/extensions/paho-mqtt5.html) (native)
* [Stitch](/camel-quarkus/latest/reference/extensions/stitch.html) (JVM only)
```

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+These three extensions are now supported in native mode:
+
+* [LRA](https://camel.apache.org/camel-quarkus/latest/reference/extensions/lra.html)
+* [Nitrite](https://camel.apache.org/camel-quarkus/latest/reference/extensions/nitrite.html)
+* [Splunk](https://camel.apache.org/camel-quarkus/latest/reference/extensions/splunk.html)
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:67]:
```suggestion
* [LRA](/camel-quarkus/latest/reference/extensions/lra.html)
* [Nitrite](/camel-quarkus/latest/reference/extensions/nitrite.html)
* [Splunk](/camel-quarkus/latest/reference/extensions/splunk.html)
```

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  since long, so you can remove them safely making sure that your application depends on `camel-quarkus-core`
+  either directly or indirectly.
+* Similarly, `camel-quarkus-main` is now a part of `camel-quarkus-core` and you can remove it safely making
+  sure that your projects depends on `camel-quarkus-core` at least transitively.
+
+## Breaking changes:
+
+The following deprecated extensions were removed in this release:
+
+* WebSocket JSR 356 (replaced by [Vert.x WebSocket](https://camel.apache.org/camel-quarkus/latest/reference/extensions/vertx-websocket.html))
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:83]:
```suggestion
* WebSocket JSR 356 (replaced by [Vert.x WebSocket](/camel-quarkus/latest/reference/extensions/vertx-websocket.html))
```

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+* All AWS SDK v1 extensions (replaced by their AWS SDK v2 counterparts)
+
+A part of the functionality originally present in `camel-quarkus-xml-io` has moved to
+`camel-quarkus-xml-io-dsl`. Hence when you see e.g. `Cannot find RoutesBuilderLoader in classpath supporting file extension: xml` in your log, you'll need to replace `camel-quarkus-xml-io` with `camel-quarkus-xml-io-dsl`.
+
+
+## Full Changelog of Camel Quarkus 1.8.0
+
+* [Fixed issues](https://github.com/apache/camel-quarkus/milestone/12?closed=1)
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:92]: You could point to a release also, something like:

```suggestion
* [Fixed issues](https://camel.apache.org/releases/q-1.8.0/)
```

That would require creating a `release-1.8.9.md` file here: https://github.com/apache/camel-website/tree/master/content/releases/q

**Review by zregvart — CHANGES_REQUESTED** [review]
Some comments on the use of absolute links to camel.apache.org and suggestion to link to the release note.

**Comment by ppalaga** [comment]
> Some comments on the use of absolute links to camel.apache.org

Ups, sorry, I actually wanted to remove them before pushing.

**Comment by ppalaga** [comment]
b8b4c56 adopts the suggestions (thanks!) of @zregvart

**Comment by ppalaga** [comment]
dcc8775 : fixed some glitches after seeing the preview

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+* WebSocket JSR 356 (replaced by [Vert.x WebSocket](/camel-quarkus/latest/reference/extensions/vertx-websocket.html))
+* All AWS SDK v1 extensions (replaced by their AWS SDK v2 counterparts)
+
+A part of the functionality originally present in `camel-quarkus-xml-io` has moved to
+`camel-quarkus-xml-io-dsl`. Hence when you see e.g. `Cannot find RoutesBuilderLoader in classpath supporting file extension: xml` in your log, you'll need to replace `camel-quarkus-xml-io` with `camel-quarkus-xml-io-dsl`.
+
+
+## Full Changelog of Camel Quarkus 1.8.0
+
+* [Fixed issues](https://camel.apache.org/releases/q-1.8.0/)
```
Comment (zregvart) [inline content/blog/2021/03/camel-quarkus-release-1.8.0/index.md:90]: One more here:

```suggestion
* [Fixed issues](/releases/q-1.8.0/)
```

**Review by apupier — APPROVED** [review]

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md (hunk 8 above)
Comment (ppalaga) [reply content/blog/2021/03/camel-quarkus-release-1.8.0/index.md]: Sorry, I am unteachable :/

**Comment by ppalaga** [comment]
89c3b7a removes the last `https://camel.apache.org`

**Review by zregvart — APPROVED** [review]
I think we should be okay now. Thanks!

File: content/blog/2021/03/camel-quarkus-release-1.8.0/index.md (hunk 8 above)
Comment (zregvart) [reply content/blog/2021/03/camel-quarkus-release-1.8.0/index.md]: That's why we have checks, I myself recently did the exact same mistake on another PR...

**Review by aldettinger — APPROVED** [review]

**Pull request merged**